lib/ex_doc/utils.ex

```diff
@@ -1,6 +1,14 @@
 defmodule ExDoc.Utils do
   @moduledoc false
 
+  def run_before_closing_body_tag(%{before_closing_body_tag: {m, f, a}}, module) do
+    apply(m, f, a ++ [module])
```

Comment:
```suggestion
    apply(m, f, [module | a])
```

Typically we prepend args to MFAs.

Overall this looks good to me!